research-software-directory/RSD-as-a-service
# Use form for personal and organisation signup

# Use form input for personal and organisation signup

Changes proposed in this pull request:
*   Instead of directly linking to opening an email client we first present a form with number of inputs to user in order to give some guidance which information should be included in the request email.
*   Note! At this point there are no "hard" validations, the user is able to use send email button at any time 

How to test:
* `make start` to rebuild all
* Navigate to home page.
* Click on `Sign up to contribute` button, the form should appear (example 1)
* Provide info and click on the button 'Send email", the email application should open with concept email containing all info
* Click on `Register your organisation`, the form should appear (example 1)
* Provide info and click on the button 'Send email", the email application should open with concept email containing all info
* Cancel should also work.
* The form will take complete screen on mobile

## 1. Personal Sign Up Form

![image](https://user-images.githubusercontent.com/9204081/202145321-18bf4caf-8021-44c2-b24f-4478eee59616.png)

## 2. Organisation Sign Up Form
![image](https://user-images.githubusercontent.com/9204081/202145430-9a19ec7f-e522-4004-925a-c8d861d5b796.png)

PR Checklist:
